services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Persistence.kt

```diff
@@ -211,6 +206,22 @@ class HankeEntity(
         }
     }
 
+    fun alkuPvm(): ZonedDateTime? {
```

Comment:
Map and filterNotNull could be combined to mapNotNull.

```
        listOfHankeAlueet
            .mapNotNull { it.haittaAlkuPvm }
            .minOfOrNull { it }?.atStartOfDay(TZ_UTC)
```

Reply:
c